templates/satspay/display.html

```diff
@@ -249,6 +371,12 @@
           hostname += '/testnet'
         }
         return hostname
+      },
+      unifiedQR() {
+        let bitcoin = this.charge.onchainaddress && this.charge.onchainaddress.toUpperCase()
+        let lightning = this.charge.payment_request && this.charge.payment_request.toUpperCase()
+        let amount = this.charge.amount / 1e8
+        return `bitcoin:${bitcoin}?amount=${amount}&lightning=${lightning}`
```

Comment:
- what does it show if `${bitcoin}` or `${lightning}` are undefined?
   - is that in accordance with the BIP21 spec?

Reply:
according to the spec (https://bitcoinqr.dev/) it should show just the bitcoin address... i'm still figuring it out... focusing on getting the markup nicer.